README.md

```diff
@@ -5,246 +5,136 @@
 
 Apollo-Android is a GraphQL compliant client that generates Java models from standard GraphQL queries.  These models give you a typesafe API to work with GraphQL servers.  Apollo will help you keep your GraphQL query statements together, organized, and easy to access from Java. Change a query and recompile your project - Apollo code gen will rebuild your data model.  Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection (see example generated code below).  Future versions of Apollo-Android will also work with AutoValue and other value object generators.
 
+## Adding Apollo to your Project
 
+The latest Gradle plugin version is 0.2.2.
 
-
-## Usage
-Add the following to your project's top level `build.gradle` file 
+To use this plugin, add the dependency to your project's build.gradle file:
 
 ```groovy
-
-ext {
-    compileSdkVersion = 25
-    buildToolsVersion = '25.0.2'
-    apolloVersion = '0.2.1'
-
-    dep = [
-            androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta3',
-            apolloPlugin: "com.apollographql.android:gradle-plugin:$apolloVersion",
-            apolloConverter: "com.apollographql.android:converter:$apolloVersion",
-            retrofit: 'com.squareup.retrofit2:retrofit:2.1.0']
-}
-
-subprojects {
-    buildscript {
-        repositories {
-            jcenter()
-            maven { url "https://jitpack.io" }
-            maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
-        }
-    }
+buildscript {
     repositories {
         jcenter()
-        maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+        maven { url "https://jitpack.io" }
+        mavenCentral()
+    }
+    dependencies {
+        
+        classpath 'com.apollographql.android:gradle-plugin:0.2.2'
     }
 }
-
-
 ```
 
-within your app module's `build.gradle` add the following:
-```
-buildscript {
-    dependencies {
-        classpath dep.androidPlugin
-        classpath dep.apolloPlugin
-    }
+Latest development changes are available in Sonatype's snapshots repository:
 
+```groovy
+buildscript {
+  repositories {
+    jcenter()
+    maven { url "https://jitpack.io" }
+    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+  }
+  dependencies {
+    
+    classpath 'com.apollographql:apollo-gradle-plugin:0.2.3-SNAPSHOT'
+  }
 }
+```
 
-apply plugin: 'com.android.application'
-apply plugin: 'com.apollographql.android'
+The plugin can then be applied as follows within your app module's `build.gradle` :
 
-dependencies {
-    dep.apolloConverter
-    dep.retrofit
-}
+apply plugin: 'com.apollographql.android'
```

Comment:
can we add this line below

`The Android Plugin must be applied before the Apollo plugin`